### DISCUSSION TITLE
How to use the ml-energy without docker and using rocm

### DISCUSSION THREAD

**Discussion by Hobbes-Le-Chat** [ml-energy/leaderboard-v2#36] [Q&A]
Hello, 

I would like to use the ML script to test one of the models I trained. Is there any documentation available on how to use ML-Energy without Docker and with ROCm support?

Thanks

**Comment by jaywonchung** [comment]
Hey @Hobbes-Le-Chat,

If you have your own model that you want to benchmark, I would rather suggest you to use the Zeus library, which is basically what we're using to measure time and energy. See details [here](https://ml.energy/zeus/measure/). You don't need Docker for this.

The tricky part is about ROCm. We're currently actively working on adding support for AMD GPUs but due to a bug in AMD's system management library, we're not really sure when we'd be able to officially land support. The tracking GitHub issue is [this](https://github.com/ml-energy/zeus/issues/22), which you can subscribe to get updates on its status.
